documentation/book/ref-kafka-logging.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-logging.adoc
+
+[id='kafka-logging-{context}']
+= Loggers
+
+In the Strimzi several components are running. Each of these components has it's own loggers which values can be set.
```

Comment:
Please change it to "{ProductName} consists of several components. Each component has its own loggers and is configurable. This section provides information about loggers of various components."